packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -119,6 +120,29 @@ const LineItemMeta = styled.div< { theme?: Theme } >`
 	gap: 2px 10px;
 `;
 
+const UpgradeCreditInformationLineItem = styled( LineItemMeta )< { theme?: Theme } >`
+	justify-content: flex-start;
+	gap: 4px;
+	.gridicon {
+		fill: ${ ( props ) => props.theme.colors.textColorLight };
+	}
+`;
+
+const UpgradeCreditPopover = styled( Popover )< { theme?: Theme } >`
+	&.popover .popover__inner {
+		color: ${ ( props ) => props.theme.colors.textColorOnDarkBackground };
```

Comment:
This feels a little weird that we try to set `theme` as optional, yet `props.theme.colors` is valid (and the engine sees it as `(property) theme: Theme & Theme` cause it's coming from `styled`). I think we don't need to define `< { theme?: Theme } >` above 🤔

Reply:
never mind obviously. just a little weird

Reply:
I see what you mean! There is no guarantee that `props.theme.colors` would exist in the JS world, so I guess `< { theme?: Theme } >` just acts as an hint for TS (and maybe editors).